src/invisible/getting-started/ecosystem-overview.md

```diff
@@ -4,4 +4,57 @@ description: TODO
 
 # Ecosystem Overview
 
-Coming soon!
\ No newline at end of file
+Once you have made a recording using your Pupil Invisible glasses, the next step is to transfer it off of the phone for further processing.
+We will introduce the different available options in the article alongside a couple other key components.
+
+## Pupil Cloud
+Pupil Cloud is our cloud-based storage and analysis platform and is the recommended tool for handling your Pupil Invisible recordings.
+It makes it easy to store all your data securely in one place and it offers a variety of options for analysis.
+
+<div class="pb-4" style="display:flex;justify-content:center;">
+  <v-img
+    :src="require('../../media/cloud/imgs/cloud-capture-store-analyze.jpg')"
```

Comment:
This image could be clearer with titles: while it's obvious that the first image - "capture" - shows the glasses, the difference between "upload" (Companion App) and "analyze" (Cloud) might be less clear. I suggest adding titles like "PI Glasses", "PI Companion App", and "Pupil Cloud" to the images.